test_pull_requests

```diff
@@ -15,12 +15,12 @@
 
 require 'rubygems'
 require 'uri'
+require 'cgi'
```

Comment:
Moved this because `hub` aliases the `CGI` module with one that lacks `escapeHTML`